front/app/components/admin/ReportExportMenu/index.tsx

```diff
@@ -52,13 +52,41 @@ export interface ReportExportMenuProps {
   currentProjectFilterLabel?: string | undefined;
   currentGroupFilterLabel?: string | undefined;
   currentTopicFilterLabel?: string | undefined;
-  xlsxData?: XlsxData;
+  xlsx?: XlsxConfig;
```

Comment:
Changed the prop structure a bit, so that it's clear that using an endpoint, data object, or download function (which I just added) are mutually exclusive